lib/bindings/HTTPBinding.js

```diff
@@ -127,7 +127,9 @@ function parseData(req, res, next) {
         next(error);
     } else {
         req.jsonPayload = data;
-
+        if (req.body !== undefined ) {
+            data = data.toString('hex');
+        }
```

Comment:
Not sure what is the intention here. data is just after used in a log, but req.jsonPayload has previos data, no data.toString('hex').
IMHO these lines are doing nothing; maybe should be rewritten.

Reply:
@Keshav-NEC could you reply to this?

Reply:
The lines are written for test case in command-polling-test.js. https://github.com/telefonicaid/iotagent-json/blob/952447fe8abdcf8cb670a5fcf06b42be28d3b30e/test/unit/ngsiv2/commandsPolling-test.js#L182 
Please refer to the conversation https://github.com/telefonicaid/iotagent-json/pull/736#discussion_r1295483708